src/components/lab/canvas/Mountains.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Line, Group } from 'react-konva';
+import {
+  generateFullMountainPoints,
+  generateHalfMountainPoints,
+} from '../../../utils/canvas';
+import {
+  DEFAULT_ICE_COVER,
+  FULL_MOUNTAIN_HEIGHT,
+  FULL_MOUNTAIN,
+  FULL_MOUNTAIN_WIDTH,
+  FULL_MOUNTAIN_X_INDENT,
+  HALF_MOUNTAIN_HEIGHT,
+  HALF_MOUNTAIN,
+  HALF_MOUNTAIN_WIDTH,
+  MOUNTAINS_INDENT_Y,
+  MOUNTAIN_FILL,
+  MOUNTAIN_LINES_TENSION,
+} from '../../../config/constants';
+import IceCover from './IceCover';
+
+const Mountains = ({
+  stageWidth,
+  groundHeight,
+  groundWidth,
+  groundBeginsY,
+}) => {
+  const fullMountainWidth = FULL_MOUNTAIN_WIDTH * groundWidth;
```

Comment:
I think it would be clearer if you add `FACTOR` to your constants to opposite pixel defined size.

Reply:
Do you mean something like defining `DEFAULT_STAGE_WIDTH` and `DEFAULT_STAGE_HEIGHT`, and then all the canvas elements scale with respect to those? I.e. we determine an element's size with something like `DEFAULT_ELEMENT_SIZE * currentWidth / DEFAULT_STAGE_WIDTH`. I think that might be a good idea. In general let's reconsider how the responsive-ness is working and discuss...